packages/tokens/src/colors.ts

```diff
@@ -0,0 +1,300 @@
+      },
+      inverse: {
+        default: gray.light2,
+        hover: gray.light3,
+        focus: blue.light2,
+      },
+      info: {
+        default: blue.dark3,
+        hover: blue.dark3,
+        focus: blue.dark3,
+      },
+      warning: {
+        default: yellow.dark3,
+        hover: yellow.dark3,
+        focus: yellow.dark3,
+      },
+      success: {
+        default: green.dark3,
+        hover: green.dark3,
+        focus: green.dark3,
+      },
+      error: {
+        default: red.dark3,
+        hover: red.dark3,
+        focus: red.dark3,
+      },
+      disabled: {
+        default: gray.dark2,
+        hover: gray.dark2,
+        focus: gray.dark2,
+      },
+    },
+    border: {
+      primary: {
+        default: gray.base,
+        hover: gray.base,
+        focus: blue.light1,
+      },
+      secondary: {
+        default: gray.dark2,
+        hover: gray.dark2,
+        focus: blue.light1,
+      },
+      success: {
+        default: green.dark1,
+        hover: green.dark1,
+        focus: blue.light1,
+      },
+      error: {
+        default: red.light1,
+        hover: red.light1,
+        focus: blue.light1,
+      },
+      disabled: {
+        default: gray.dark2,
+        hover: gray.dark2,
+        focus: gray.dark2,
+      },
+    },
+    icon: {
+      primary: {
+        default: gray.light1,
+        hover: gray.light3,
+        focus: blue.light3,
+      },
+      secondary: {
+        default: gray.base,
+        hover: gray.light3,
+        focus: blue.light3,
+      },
+      inverse: {
+        default: white,
+        hover: black,
+        focus: blue.dark2,
+      },
+      info: {
+        default: blue.light1,
+        hover: blue.light1,
+        focus: blue.light1,
+      },
+      warning: {
+        default: yellow.base,
+        hover: yellow.base,
+        focus: yellow.base,
+      },
+      success: {
+        default: green.base,
+        hover: green.base,
+        focus: green.base,
+      },
+      error: {
+        default: red.light1,
+        hover: red.light1,
+        focus: red.light1,
+      },
+      disabled: {
+        default: gray.dark1,
+        hover: gray.dark1,
+        focus: gray.dark1,
+      },
+    },
+    text: {
+      primary: {
+        default: gray.light2,
+        hover: gray.light2,
+        focus: blue.light3,
+      },
+      secondary: {
+        default: gray.light1,
+        hover: gray.light2,
+        focus: blue.light3,
+      },
+      inversePrimary: {
+        default: black,
+        hover: black,
+        focus: blue.dark2,
+      },
+      inverseSecondary: {
+        default: gray.dark2,
+        hover: black,
+        focus: blue.dark2,
+      },
+      error: {
+        default: red.light1,
+        hover: red.light1,
+        focus: red.light1,
+      },
+      disabled: {
+        default: gray.dark1,
+        hover: gray.dark1,
+        focus: gray.dark1,
+      },
+    },
+  },
+  [Mode.Light]: {
+    bg: {
```

Comment:
this block sorts differently from the others. can we make them consistently `default`, `hover`, `focus`?

Reply:
yes updated